packages/app-cli/app/command-rmnote.ts

```diff
@@ -31,13 +31,14 @@ class Command extends BaseCommand {
 
 		let ok = true;
 		if (!force && notes.length > 1) {
-			ok = await this.prompt(_('%d notes match this pattern. Delete them?', notes.length), { booleanAnswerDefault: 'n' });
+			ok = await this.prompt(_n('%d note matches this pattern. Delete it?', '%d notes match this pattern. Delete them?', notes.length, notes.length), { booleanAnswerDefault: 'n' });
 		}
 
 		const permanent = (args.options?.permanent === true) || notes.every(n => !!n.deleted_time);
 		if (!force && permanent) {
 			const message = (
-				notes.length === 1 ? _('This will permanently delete the note "%s". Continue?', notes[0].title) : _('%d notes will be permanently deleted. Continue?', notes.length)
+				notes.length === 1 ? _('This will permanently delete the note "%s". Continue?', notes[0].title) :
+					_n('%d note will be permanently deleted. Continue?', '%d notes will be permanently deleted. Continue?', notes.length, notes.length)
```

Comment:
I don't think we should have so many cases for this. Just keep the `_n` probably.

```suggestion
				_n('%d note will be permanently deleted. Continue?', '%d notes will be permanently deleted. Continue?', notes.length, notes.length)
```